bin/qcodes_logger_server

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+import logging
+import zmq
+
+
+def broker(frontend_addres="tcp://*:5559", backend_address="tcp://*:5560"):
```

Comment:
Maybe use a non-standard ports here? Just to avoid interference with other programs?
Ports from qcodes config? (I know this is independent of qc..)